avm/res/resources/deployment-script/tests/e2e/max/main.test.bicep

```diff
@@ -0,0 +1,74 @@
+targetScope = 'subscription'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'avm-${namePrefix}-resources.deploymentscripts-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param location string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'rdsps'
+
+@description('Optional. Enable telemetry via a Globally Unique Identifier (GUID).')
+param enableTelemetry bool = true
+
+@description('Optional. A token to inject into the name of each resource.')
+param namePrefix string = 'avmds'
+
+@maxLength(5)
+@description('Optional. Random Guid to be using in naming prefix.')
+param namingGuid string = toLower(substring(newGuid(), 0, 5))
```

Comment:
```suggestion
```
Please remove. The name will be unique as per the NamePrefix secret you set up in your fork.